spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -729,4 +729,11 @@ def get_decision_product(resp)
       end
     end
   end
+
+  describe '#pii_from_doc' do
```

Comment:
I tried running this spec on `main`, and it passed. Is there any test we could write that would fail on `main` but pass due to these changes? If not that's okay. I thought it might help me wrap my head around this change, as well as keep a similar thing from happening again. (Maybe this case doesn't warrant it though)